### DISCUSSION TITLE
[Bug]: unable to download the voxlingua107 dataset

### DISCUSSION THREAD

**Discussion by cc-cherie** [speechbrain/speechbrain#1924] [Q&A] [answered]
### Describe the bug

The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

### Expected behaviour

download the Voxlingua107

**Comment by asumagic** [comment]
I went ahead and migrated the issue to a discussion as it seems you are having an issue with accessing <http://bark.phon.ioc.ee/voxlingua107/>, which is external to the SpeechBrain project and not hosted by it. I can confirm that the link does not work for me anymore either.

Looking at the Wayback Machine, it seems that this URL returned error code 404 before failing, so I don't know if it'll ever be available through there again. I am not sure how one could download it nowadays.

EDIT: actually, we link to it, so it might be still good to figure out... But if the official source is down I'm not sure what to do.

**Comment by Gastron** [comment, marked as answer]
I also noticed this, and I have informed @alumae, who is one of the authors of this dataset. I'm sure I can relay this information: the bark.phon.ioc.ee server is down due to hardware problems, but the intention is to fix it.

**Reply by cc-cherie** [reply]
That's great！ Thank you.

**Comment by alumae** [comment]
We had a hardware failure which is fixed now. https://bark.phon.ioc.ee/voxlingua107/
